pkg/kafka/cluster_admin_client_mock_impl.go

```diff
@@ -26,13 +26,26 @@ const (
 	defaultMockControllerID = 1
 )
 
-var (
-	// defaultMaxMessageBytes specifies the default max message bytes.
-	defaultMaxMessageBytes = "10485760"
-	// defaultMaxMessageBytes specifies the default min insync replicas for broker and default topic.
+const (
+	// defaultBrokerMessageMaxBytes specifies the broker's `message.max.bytes`, default to 1048576.
+	defaultBrokerMessageMaxBytes = "1048576"
+
+	// defaultTopicMaxMessageBytes specifies the topic's `max.message.bytes`, default to 1048576.
+	defaultTopicMaxMessageBytes = "1048576"
```

Comment:
I don't think we need such identical two values?

Reply:
one for broker's default `message.max.bytes`, one for topic's `max.message.bytes`.

Is there any idea?

Reply:
They are all the same value? And still in the mock?

Reply:
yes, the default value is the same but has a different meaning. so I think to keep 2 different is necessary, to make it more clear.

Reply:
Can't see what's different about it? There should be only one default value for ticdc. For this mock they are the same value. There is no need to waste memory storing it twice.

Reply:
The same default value has been copied at least 4 times here.

Reply:
This defaultBrokerMessageMaxBytes looks useless except that it has been copied once.